### PR TITLE
[TRANSLATION] Improve Chinese translation

### PR DESCRIPTION
**Pull request by njlyf2011**
1. Changed some wording of these files according to the translation requirements.
2. Fixed some mistakes.
3. Change my name.

### CODE REVIEW

**Comment by kernelbin** [comment]
Translations & corrections part LGTM.

File: base/applications/network/ping/lang/zh-CN.rc [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * PROJECT:     ReactOS Ping Command
  * LICENSE:     MIT (https://spdx.org/licenses/MIT)
  * PURPOSE:     Chinese (Simplified) resource file
- * TRANSLATORS: Copyright 2018 Luo Yufan <njlyf2011@hotmail.com>
+ * TRANSLATORS: Copyright 2018  <njlyf2011@hotmail.com>
```
Comment (binarymaster) [inline base/applications/network/ping/lang/zh-CN.rc:5]:
```suggestion
 * TRANSLATORS: Copyright 2018 Luke Luo <njlyf2011@hotmail.com>
```

File: dll/cpl/inetcpl/lang/zh-CN.rc [hunk 2]
```diff
@@ -20,7 +20,7 @@
  */
 
 /* Simplified Chinese translation by Henry Tang Ih 2015 (henrytang2@hotmail.com) */
-/* Translation has been changed by Luo Yufan 2018 (njlyf2011@hotmail.com) */
+/* Translation has been updated by Luke Luo 2022 (njlyf2011@hotmail.com) */
```
Comment (binarymaster) [inline dll/cpl/inetcpl/lang/zh-CN.rc:23]:
```suggestion
/* Translation has been updated by Luke Luo 2018-2022 (njlyf2011@hotmail.com) */
```

File: modules/rosapps/applications/sysutils/systeminfo/lang/zh-CN.rc [hunk 3]
```diff
@@ -1,6 +1,11 @@
-LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
+/*
+ * PROJECT:     System Information
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Chinese (Traditional) resource file
```
Comment (binarymaster) [inline modules/rosapps/applications/sysutils/systeminfo/lang/zh-CN.rc:4]: zh-CN is simplified, not traditional
```suggestion
 * PURPOSE:     Chinese (Simplified) resource file
```

File: base/applications/rapps/lang/zh-CN.rc [hunk 4]
```diff
@@ -1,6 +1,13 @@
-/* Translated by Song Fuchang (0xfc) <sfc_0@yahoo.com.cn> */
-/* Imporved by Li Keqing <auroracloud4096@gmail.com>, Luo Yufan (njlyf2011@hotmail.com) */
-/* Improved by He Yang <1160386205@qq.com> */
+/*
+ * PROJECT:     ReactOS Applications Manager
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Chinese (Traditional) resource file
```
Comment (binarymaster) [inline base/applications/rapps/lang/zh-CN.rc:4]:
```suggestion
 * PURPOSE:     Chinese (Simplified) resource file
```

**Review by binarymaster — COMMENTED** [review]
Some minor fixes. Need to ask the developer team whether it's allowed to change the name for older copyrights (like, from previous years).

File: base/system/chkdsk/lang/zh-CN.rc [hunk 5]
```diff
@@ -1,4 +1,4 @@
-/* Translation has been created by Luo Yufan in 2018 (njlyf2011@hotmail.com) */
+/* Translation has been created by Luke Luo in 2018 (njlyf2011@hotmail.com) */
```
Comment (binarymaster) [inline base/system/chkdsk/lang/zh-CN.rc:1]:
```suggestion
/* Translation has been created by Luke Luo in 2018 <njlyf2011@hotmail.com> */
```

File: base/system/subst/lang/zh-CN.rc [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /* Simplified Chinese translation by Song Fuchang (0xfc) <sfc_0@yahoo.com.cn> 2011 */
-/* Translation has been changed by Luo Yufan (njlyf2011@hotmail.com) 2018 */
+/* Translation has been changed by Luke Luo (njlyf2011@hotmail.com) 2018 */
```
Comment (binarymaster) [inline base/system/subst/lang/zh-CN.rc:2]:
```suggestion
/* Translation has been changed by Luke Luo <njlyf2011@hotmail.com> 2018 */
```

File: dll/cpl/openglcfg/lang/zh-CN.rc [hunk 7]
```diff
@@ -1,4 +1,4 @@
-/* Translation has been created by Luo Yufan in 2018 (njlyf2011@hotmail.com) */
+/* Translation has been created by Luke Luo in 2018 (njlyf2011@hotmail.com) */
```
Comment (binarymaster) [inline dll/cpl/openglcfg/lang/zh-CN.rc:1]:
```suggestion
/* Translation has been created by Luke Luo <njlyf2011@hotmail.com> in 2018 */
```

File: dll/shellext/devcpux/lang/zh-CN.rc [hunk 8]
```diff
@@ -1,4 +1,4 @@
-/* Translated by Luo Yufan (njlyf2011@hotmail.oom) on 2018-11-23 */
+/* Translated by Luke Luo (njlyf2011@hotmail.oom) on 2018-11-23 */
```
Comment (binarymaster) [inline dll/shellext/devcpux/lang/zh-CN.rc:1]:
```suggestion
/* Translated by Luke Luo <njlyf2011@hotmail.com> on 2018-11-23 */
```

File: dll/win32/userenv/lang/zh-TW.rc [hunk 9]
```diff
@@ -1,4 +1,4 @@
-/* Translation has been created by Luo Yufan (njlyf2011@hotmail.com) */
+/* Translation has been created by Luke Luo (njlyf2011@hotmail.com) */
```
Comment (binarymaster) [inline dll/win32/userenv/lang/zh-TW.rc:1]:
```suggestion
/* Translation has been created by Luke Luo <njlyf2011@hotmail.com> */
```

File: modules/rosapps/applications/sysutils/fontsub/lang/zh-CN.rc [hunk 10]
```diff
@@ -11,7 +11,7 @@
 /* UTF-8 */
 #pragma code_page(65001)
 
-/* Translated by Luo Yufan (njlyf2011@hotmail.com) on 2018-11-13 */
+/* Translated by Luke Luo (njlyf2011@hotmail.com) on 2018-11-13 */
```
Comment (binarymaster) [inline modules/rosapps/applications/sysutils/fontsub/lang/zh-CN.rc:14]:
```suggestion
/* Translated by Luke Luo <njlyf2011@hotmail.com> on 2018-11-13 */
```

File: modules/rosapps/applications/sysutils/systeminfo/lang/zh-CN.rc [hunk 11]
```diff
@@ -1,6 +1,11 @@
-LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
+/*
+ * PROJECT:     System Information
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Chinese (Simplified) resource file
+ * TRANSLATORS: Copyright 2018, 2022 Luke Luo (njlyf2011@hotmail.com)
```
Comment (binarymaster) [inline modules/rosapps/applications/sysutils/systeminfo/lang/zh-CN.rc:5]:
```suggestion
 * TRANSLATORS: Copyright 2018, 2022 Luke Luo <njlyf2011@hotmail.com>
```

File: base/applications/rapps/lang/zh-CN.rc [hunk 12]
```diff
@@ -1,6 +1,13 @@
...
-/* Imporved by Li Keqing <auroracloud4096@gmail.com>, Luo Yufan (njlyf2011@hotmail.com) */
-/* Improved by He Yang <1160386205@qq.com> */
+/*
+ * PROJECT:     ReactOS Applications Manager
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Chinese (Simplified) resource file
+ * TRANSLATORS: Copyright Song Fuchang (0xfc) <sfc_0@yahoo.com.cn>
+ *              Copyright Li Keqing <auroracloud4096@gmail.com>
+ *              Copyright Luke Luo (njlyf2011@hotmail.com)
+ *              Copyright He Yang <1160386205@qq.com>
```
Comment (binarymaster) [inline base/applications/rapps/lang/zh-CN.rc:8]:
```suggestion
 * TRANSLATORS: Copyright 2011 Song Fuchang (0xfc) <sfc_0@yahoo.com.cn>
 *              Copyright 2013 Yuntian Zhang <saberconer@gmail.com>
 *              Copyright 2017-2018 Li Keqing <auroracloud4096@gmail.com>
 *              Copyright 2018, 2022 Luke Luo <njlyf2011@hotmail.com>
 *              Copyright 2020 He Yang <1160386205@qq.com>
```

**Pull request merged**